dashboard/client/src/pages/overview/cards/RecentJobsCard.tsx

```diff
@@ -32,96 +18,40 @@ type RecentJobsCardProps = {
   className?: string;
 };
 
+const getLink = (job: UnifiedJob) => {
+  if (job.job_id !== null && job.job_id !== "") {
+    return `?/jobs/${job.job_id}`;
```

Comment:
why does this start with `?/`?

Reply:
Good catch, I think I made this typo after the manual testing. I just tested the Jobs Card again


Uploading Screen Recording 2023-05-05 at 12.06.23 PM.mov…